src/taxpasta/infrastructure/application/metaphlan/metaphlan_profile_standardisation_service.py

```diff
@@ -38,7 +38,7 @@ class MetaphlanProfileStandardisationService(ProfileStandardisationService):
     """Define a standardisation service for metaphlan profiles."""
 
     # Metaphlan only reports up to six decimals so this number should be large enough.
-    LARGE_INTEGER = int(1e6)
+    LARGE_INTEGER = 1_000_000
```

Comment:
What is that exactly?

Reply:
Just a different way to write the same thing. I thought that it's maybe slightly easier to read.